### ISSUE TITLE
[Request]: Show reply-to post & media when editing alt in composer

### ISSUE THREAD

**Issue by GeopJr** [GeopJr/Tuba#320]
### Describe the request

Post you are replying to should be visible in the composer. Media attachment you are adding alt text to should also be visible.

### Implementation Details

- [ ] This should be an option in settings.
- [ ] This should be only available to some fediverse backends. (Include which ones on the above field).
- [ ] This is client-only (and shouldn't sync with the instance).
- [X] This follows the [GNOME HIG](https://developer.gnome.org/hig/).

**Comment by eobet** [comment]
I hope this issue includes re-designing the fact that the alt text pop up currently entirely blocks the media you are trying to add a description for. ;)

**Comment by GeopJr** [comment]
It will!
